FrontEnd/styles/package.scss

```diff
@@ -69,12 +69,17 @@
                 display: block;
                 font-size: 11px;
             }
-
+            
```

Comment:
Nit: Let's not add whitespace like this. The easiest way to fix this is to set these two options in Xcode's Editing settings.

![Screenshot 2024-09-12 at 12 59 36@2x](https://github.com/user-attachments/assets/74a19a18-81a7-4c5a-83a7-67e9e710fc5a)